tools/clang/unittests/HLSLExec/ExecutionTest.cpp

```diff
@@ -10248,9 +10242,9 @@ void ExecutionTest::WaveSizeTest() {
 bool AtomicResultMatches(const BYTE *uResults, uint64_t gold, size_t size) {
   if (memcmp(uResults, &gold, size)) {
     if (size == 4)
-      LogCommentFmt(L"  value %d is not %d", ((uint32_t*)uResults)[0], (uint32_t)gold);
+      LogCommentFmt(L"  value %d is not %d", ((const uint32_t*)uResults)[0], (uint32_t)gold);
```

Comment:
Just above you used a const_cast for casting a non-const pointer to const. I'm not sure it makes any difference, but it would be nice to be consistent. I don't feel this strongly though.

Reply:
The const_cast is casting a const to non-const pointer for CreateBitmapForMemory is which only accept non-const pointer.

Here we don't remove the const, but cast from const BYTE* to const uint32_t*.